src/spawn_sync.cc

```diff
@@ -683,7 +683,9 @@ Local<Object> SyncProcessRunner::BuildResultObject() {
   if (term_signal_ > 0)
     js_result->Set(context, env()->signal_string(),
                    String::NewFromUtf8(env()->isolate(),
-                                       signo_string(term_signal_))).FromJust();
+                                       signo_string(term_signal_), v8::NewStringType::kNormal)
```

Comment:
Does this hit the 80 char limit?

Reply:
It does, dunno how autoformatting left this one out. Will do.